spec/routing/notes_routing_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
```

Comment:
We do not test routes.rb in real projects, but you can leave it since it's a pet project.